lib/praxis/routing_config.rb

```diff
@@ -49,11 +49,14 @@ def any(path, opts={})     add_route 'ANY',     path, opts end
 
     def add_route(verb, path, options={})
       unless path =~ ABSOLUTE_PATH_REGEX
-        path = prefix + path        
+        path = prefix + path
       end
 
       path = (base + path).gsub('//','/')
-      pattern = Mustermann.new(path)
+      mustermann_options = {ignore_unknown_options: true}
+      # Reject our own options
+      possible_mustermann_options = options.reject{|(k,_)| [:name].include?(k) }
+      pattern = Mustermann.new(path, mustermann_options.merge( possible_mustermann_options ))
```

Comment:
because I don't want to make that show in our `Route` options (just want to pass it down to Mustermann).